schemas/channel.graphql

```diff
@@ -13,7 +13,7 @@ type Channel {
   type: ChannelType
   name: String
   messages: [Message]
-  membership: Membership
+  membership: [Membership]
```

Comment:
This should be reverted. [Channel] doesn't have many [Membership].

Reply:
Sorry I was confused. Yes, we should have to get many memberships from the [Channel] (we need to know who is in the channel).
However, it looks like we need additional queries for `membership`.
What about adding both `memberships` and `membership` or `memberships` and  `myMembership`?

Reply:
I changed membership to memberships (plural). However, I am confused about the usage of what you mentioned as `membership` or `myMembership`. Is it for querying the channel's owner membership? @hyochan

Reply:
It's for querying my membership associated with `Channel`.
When you are querying `Channel`, you'd want to get `myMembership` together at one resolver!
Please tell me when it still isn't clear!

Reply:
Ah ha! I will work on it and push commits  as soon as possible. Thanks 🙏

Reply:
I added `myMembership` fragment query resolver in Channel resolver. I wanted not to call `getUser` again in fragment query resolver. However, I could not figure out how I can remove the duplicated db query for now. Is there any other option to share data between resolver and fragment resolver?

Reply:
@impressor615 It isn't a good idea to make more threads to resolved review. Let's move it down.